src/ui/platforms/desktop/src/ui/character/characterSkin.tsx

```diff
@@ -227,12 +227,13 @@ export class CharacterSpec extends React.Component<CharacterSpecProps> {
       this.props.audioService.playSkillAudio(
         skillName,
         this.props.character.Gender,
+        Math.round(Math.random() * 1) + 1,
```

Comment:
because it has created the random index in `getSkillAudio`, it is no need to generate random index here